pkgs/development/python-modules/aioh2/default.nix

```
@@ -19,5 +19,6 @@ buildPythonPackage rec {
     description = "HTTP/2 implementation with hyper-h2 on Python 3 asyncio";
     license = licenses.bsd3;
     maintainers = [ maintainers.qyliss ];
+    broken = true;
```

do you mind adding a comment, for future maintainence
```suggestion
    broken = true; # requires h2<4
```

---

As reference: https://github.com/decentfox/aioh2/issues/24